lib/proxy/plugin_validators.rb

```diff
@@ -7,6 +7,7 @@ def initialize(plugin, setting_name)
 
     def validate!
       setting_value = @plugin.settings.send(@setting_name)
+      return true if setting_value.nil?
```

Comment:
dns_key in dns_nsupdate provider: https://github.com/theforeman/smart-proxy/blob/develop/modules/dns_nsupdate/dns_nsupdate_plugin.rb#L6.